### DISCUSSION TITLE
Connect to Amazon EC2 file directory using Filezilla and SFTP

### DISCUSSION THREAD

**Discussion by chandnie** [community/community#20666] [Discussions] [answered]
I have created an AWS EC2 Instance and I want to be able to upload files to the server directory using FileZilla in the simplest and most straightforward fashion possible.

**Comment by chiragmehta900** [comment, marked as answer]
Summary of the above video tutorial:

Edit (Preferences) > Settings > Connection > SFTP, Click "Add key file”
Browse to the location of your .pem file and select it.
A message box will appear asking for your permission to convert the file into PPK format. Click Yes, then give the file a name and store it somewhere.
If the new file is shown in the list of Keyfiles, then continue to the next step. If not, then click "Add key file..." and select the converted file.
File > Site Manager Add a new site with the following parameters:

Host: The public DNS name of your EC2 instance, or the public IP address of the server.

Protocol: SFTP

Logon Type: Normal

User: From the [docs](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html): "For Amazon Linux, the default user name is ec2-user. For RHEL5, the user name is often root but might be ec2-user. For Ubuntu, the user name is ubuntu. For SUSE Linux, the user name is root. For Debian, the user name is admin. Otherwise, check with your AMI provider."

Press Connect Button - If the saving of passwords has been disabled, you will be prompted that the logon type will be changed to 'Ask for password'. Say 'OK' and when connecting, at the password prompt push 'OK' without entering a password to proceed past the dialog.

Note: FileZilla automatically figures out which key to use. You do not need to specify the key after importing it as described above.

If you use Cyberduck follow [this](http://youtu.be/hd4oL3WIPVM).

Check [this](https://stackoverflow.com/questions/10337951/amazon-ec2-ftp-write-permission) post if you have any permission issues.
